packages/shared/src/normalizeProp.ts

```diff
@@ -29,10 +29,11 @@ export function normalizeStyle(
 
 const listDelimiterRE = /;(?![^(]*\))/g
 const propertyDelimiterRE = /:(.+)/
+const styleCommentRE = /\/\*.*?\*\//g
```

Comment:
Consider [this demo](https://deploy-preview-6808--vue-sfc-playground.netlify.app/#eNpNkN1OwzAMhV/F5GaASLMJCU2lm8Qdb8BNbsbqtkH5U+J2VFXfHXedgNz5Oz4+jifxFmMx9ChKUeVzMpEgI/XxqL1xMSSCCRI2MEOTgoMNt2601/4cfCZwuYXDot9v3tHaAB8h2fpu86B9pdZxPIgLQhftiZArgKrbHafpap7nSnF1pcbHnmCQLtRoD1qwrsUq1WaATKNFxgsAUI/gA3W8pW/hUa3wYmrqSnjebuP364o6NG1HJewWdjP9t/w9zqoUB3FipX73FU9ivYN0p1h85eD5UtPi1Tcha1HClSyM77PUWnREMZdK1RhtGGVMOBi8yJf9di8ld8ncnCVnjG0Kva8Lj2RNMxanGBXLReo9GYcFZic/U7hkTBy/fn/WfhbzD4lVkds=)

```suggestion
const styleCommentRE = /\/\*.*?\*\//gs
```

Reply:
Thanks.